src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -20,6 +21,7 @@ type State = {
     areErrorsEnabled: boolean,
     conditions: Array<Object>,
     isMenuOpen: boolean,
+    selectedPrefix: string,
```

Comment:
Should this be string or `typeof WHERE | typeof AND | typeof OR`?

Reply:
It's better to be more specific, I can go with typeof instead.